src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx

```diff
@@ -65,9 +65,10 @@ You must use title case for our trademarked names including our company name plu
         <tr>
             <td>New Relic One</td>
             <td>our product/ our platform</td>
-            <td>First use: New Relic One observability platform
+            <td>First use: New Relic One (docs, UI, titles) or New Relic One observability platform (marketing content)
 
                 Subsequent uses: New Relic One or New Relic One platform
+                Note: New Relic One observability platform is recommended for marketing content where users might not be familiar with our product.
```

Comment:
```suggestion
                
                Note: New Relic One observability platform is recommended for marketing content where users might not be familiar with our product.
```

@mmfred Unless you add a line break, it all bleeds together into the previous line. (If you take a look at the preview, you'll see what I mean.